usecases/steg_asymptotic_2sensors/generate_metadata_virtual_experiment.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import yaml
+
+
+def write_metadata_to_yaml(virtual_experimental_metadata_file,
+                           a, b, c, d, e,

```

Comment:
if you want to prescribe in the experiment a, b, .. instead of Mat_B and mat_C then you could add the definition of the models here (just move that from the forward model).

Reply:
I've also changed in this correlation request to store the files in json, it's possible to keep this here as is, just want to mention this.